### ISSUE TITLE
Can the developer portal recognize the deprecated attribute in OPEN API spec 

### ISSUE THREAD

**Issue by UniperMaster** [Azure/api-management-developer-portal#1620]
### Discussed in https://github.com/Azure/api-management-developer-portal/discussions/1605

<div type='discussions-op-text'>

<sup>Originally posted by **UniperMaster** January 21, 2022</sup>
I tried setting the following attribute and it wasn't recognized, is this cater for in the development portal or can it be added to the backlog.

maybe this is general APIM requirement and needs to be raised there 

i.e

`/pet/findByTags:
get:
  deprecated: true`

</div>

**Comment by UniperMaster** [comment]
This maybe an overall API Management Service issue rather than Dev portal issue. I can currently create a Tag and display that (on an operation only) but it'd be nice if the deprecated attribute was recognized

**Comment by mikebudzynski** [comment]
> This maybe an overall API Management Service issue rather than Dev portal issue. I can currently create a Tag and display that (on an operation only) but it'd be nice if the deprecated attribute was recognized

Yes, API Management in general doesn't support the `deprecated` property. We do have plans to implement support for it, but we do not have timelines to share at this moment.

cc: @VitaliyKurokhtin

**Comment by cosminstirbu** [comment]
We're also interested in this feature.
